### ISSUE TITLE
Remove 'American English' as a name for the ASL Signbank

### ISSUE THREAD

**Issue by Woseseltops** [Signbank/Global-signbank#588]
This problem keeps popping up, so let's give it a separate issue.

Latest sighting: the public view.

**Comment by Woseseltops** [comment]
tl;dr it's complicated but fixed (fyi @ocrasborn)

Okay I've figured out what is going on here. For clarity, I will use the terms T-language for the spoken languages into which the various signs are translated (like French for the French Sign Language), and I-language for the interface language which you can use to navigate Signbank.

T-language name translations are apparently being handled by the extension django-model-translation, which automatically adds translation fields for all I-languages. In the Global Signbank we have three I-languages (English, Dutch, Chinese), so for a T-language like French we have three automatically generated T-language name fields in the admin (filled with 'French', 'Frans' and '法文'). 

The ASL Signbank uses one T-language 'American English' and one I-language 'American English', which is the reason you see this name everywhere. To change this to 'English' instead, the T-language name of 'American English' for the I-language 'American English' should just be 'English'. Normally, you can simply change this in the admin... except that django-model-translation does not work in the admin for the ASL Signbank. ( I'm not sure why, but it's probably because the code en_US is not in this list: https://github.com/django/django/blob/master/django/conf/locale/__init__.py ). Therefore, I was forced to change it in the database directly, with this simple query:

```
UPDATE dictionary_language SET name_en_US = "English" WHERE id=1
```

Finally, I used the unit tests to verify I did not break anything.
